README.md

```diff
@@ -170,6 +160,17 @@ The protocol version
 |------|------------------|------------------------------------------------------------------|-------|
 | u32 | protocol_version | The protocol version |  |
 
+#### 4 device_information
+Device information
```

Comment:
Should we add ping-protocol version here ?

Reply:
we have it already, id # 5